src/main/scad/alfons-eyckmans/aracna/eyckmans.aracna.scad

```diff
@@ -0,0 +1,55 @@
+/* ====================================================================
+
+  This is a 3D model of a mechanical puzzle. It is released under
+  the following license:
+
+  Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported
+  https://creativecommons.org/licenses/by-nc-nd/3.0/
+
+  This means the model is licensed for personal, noncommercial use
+  only. Anyone may print a copy for their own use, but selling or
+  otherwise monetizing the model or print (or any derivatives) is
+  prohibited. For details, refer to the summary at the above URL.
+
+  Puzzle design (c) Alfons Eyckmans
+  3D model (c) Tom Burns
+
+==================================================================== */
+
+include <puzzlecad.scad>
+
+require_puzzlecad_version("2.1");
+
+$burr_scale = 9;
+$burr_bevel = 1;
+$auto_layout = true;
+
+//print one copy each of spider() and sticks() below:
+
+*spider();
+*sticks();
+
+module spider() {
+  burr_piece(
+    [".x..x.|......|......|x....x|......|.x..x.",
+     ".x..x.|.xxxx.|..xx..|xxxxxx|.xxxx.|.x..x.",
+     "......|......|......|..xx..|..xx..|......",]
+  );
+}
+
+module sticks() {
```

Comment:
Maybe print "A. Eyckmans" on the inside of one of the sticks? Alfons prefers this (and I think it's generally good practice)

Reply:
Updated name formatting